### PR TITLE
Wizard Batteries

### PR DESCRIPTION
**Pull request by Kurfursten**
I haven't tested this at all.

🆑 
* rscadd: When a living, conscious wizard is above a conduction plate, it will burn him and output a large amount of power based on his relative health. You could make a wizard engine if you could hold him in place and feed him anti-burn meds.

### CODE REVIEW

**Comment by jwhitak** [comment]
~~How does this affect cyborg wizards and other horrible Bob Turnip brain transfer renditions of wizards?~~

Edit: Looks like it only checks human mobs right now ..

Edit2: 500kW per tick at full health, slowly degrading as the wizard takes damage. The 500kW is multiplied by ```Health/MaxHealth```, and they take 1 fire damage per tick (even Kelotane alone will outpace the damage).

**Comment by Kurfursten** [comment]
I fully intended for it to be easy to outpace the damage with meds. IVs are already not unlimited, so some babysitting is required.

**Comment by Theophrastus1** [comment]
holy shit we're making the lifeweb in ss13

**Comment by Eneocho** [comment]
How does the power increase if you also increase the wizard's max health using phazon? Is there a bias towards having less max health so less relative health is a smaller relative change?

**Comment by tinysaturn** [comment]
>Mogan the Destroyer attempts to remove the handcuffs!
>Mogan the Destroyer makes a loud noise.

**Comment by Kurfursten** [comment]
@Eneocho No effect, it is purely proportional to max health

**Comment by Eneocho** [comment]
>No effect, it is purely proportional to max health

Shame. I was looking forward to minmaxing the wizard battery by getting the wiz to 2x max health.

**Comment by ghost** [comment]
now we just need this 
1171137685 karno_v-engine

**Comment by Kurfursten** [comment]
Now tested and glad I did, caught some bad bugs. Should be good to go.

I also kicked up damage to 3 based on feedback - still easily counterable with meds.

**Comment by pedr69** [comment]
>3 damage per tick

seems a lot, a keloderm IV will heal one per tick
is it because the burn damage dealt is split across all limbs, and the natural regen works on each wound independently?

**Comment by Kurfursten** [comment]
Yes I noticed that I was healing naturally pretty fast when testing the plate.

**Comment by adacovsk** [comment]
> I haven't tested this at all.
> 
> 🆑
> 
> * rscadd: When a living, conscious wizard is above a conduction plate, it will burn him and output a large amount of power based on his relative health. You could make a wizard engine if you could hold him in place and feed him anti-burn meds.

At least make it so that the wizard is like a scrying orb and can leave its body if he's held in place... Seems really against fun to hold a player in one spot indefinitely

**Comment by Kurfursten** [comment]
This is how bathos wanted it. Anyway as wizzy just don't be taken alive.

**Comment by adacovsk** [comment]
> This is how bathos wanted it. Anyway as wizzy just don't be taken alive.

bathos is cringe tho. Maybe put a timer on the wizard so that he's gibbed after like 6-10 minutes

**Comment by Kurfursten** [comment]
If you just want to be gibbed anyway you can always ghost out. Same result.

**Comment by adacovsk** [comment]
> If you just want to be gibbed anyway you can always ghost out. Same result.

shouldn't incentivize locking someone up indefinitely though, that's my point

**Comment by Eneocho** [comment]
Yeah, make it so the wizman can ghost out a-la scrying orb if he's in the cuck battery for long enough (2m?), so they can at least spectate the round while keeping the whole battery deal going.

**Comment by bathosbathtime** [comment]
Per the original request:
screenie
I don't see immobility being anywhere in there so I would appreciate it that you don't call me cringe because kurf based this off of conduction plates instead of something like electropacks or similar. Call me cringe for the many actual reasons you could, like how I misspelled Arthur's name in one of the stupidest ways possible.
But, regardless, it has been coded so the debt is paid as I understand the agreement. Even if it doesn't get merged, I am satisfied. Good luck sorting out the details from here!

**Comment by adacovsk** [comment]
Increases the likelihood of wizards just murderboning as they would want to kill people hunting them for such purposes

**Comment by Kurfursten** [comment]
That's just an alternate way of writing "reduces the odds of wizstended because a cringe friendly wizard offered to join the sec team instead of doing any antagonizing"

File: code/modules/fish/fish_tank.dm [hunk 1]
```diff
@@ -800,16 +800,24 @@
...
 
 /obj/machinery/power/conduction_plate/process()
+	if(!anchored)
+		return
+	var/power = 0
 	if(check_tank())
-		var/power = 0
 		for(var/fish in attached_tank.fish_list)
 			if(fish == "electric eel")
 				power += ARBITRARILY_LARGE_NUMBER * multiplier //10000
```
Comment (Exxion) [inline code/modules/fish/fish_tank.dm:809]: Not related to this PR but what the fuck is this

**Comment by Exxion** [comment]
I also don't like the incentive to permanently lock a player up, but I appear to be wildly outvoted on this matter

File: code/modules/fish/fish_tank.dm (hunk 1 above)
Comment (Kurfursten) [reply code/modules/fish/fish_tank.dm:809]: A classic define

Comment (Exxion) [reply code/modules/fish/fish_tank.dm:809]: Classic misuse of a define, that is

**Pull request merged**